### PR TITLE
Implement parsing of 4/ group (snowdepth)

### PR DESCRIPTION
**Pull request by akrherz**

### CODE REVIEW

**Comment by phobson** [comment]
This looks a good. I'll fix the Travis script then ping you here to do a rebase.

**Comment by phobson** [comment]
ok -- a rebase from phobson/master and force push should fix this PR.

File: metar/datatypes.py [hunk 1]
```diff
@@ -263,6 +265,8 @@ def value( self, units=None ):
             return m_value/1609.344
         elif units == "FT":
             return m_value*3.28084
+        elif units == "IN":
+            return m_value*39.3701
```
Comment (phobson) [inline metar/datatypes.py]: can you write a test checking for these changes?

**Comment by akrherz** [comment]
@phobson sure thing, look better now?

**Comment by phobson** [comment]
:+1: thanks!

**Pull request merged**